### ISSUE TITLE
Structure offset accesses not involving a load or store do not use __offset(n) notation

### ISSUE THREAD

**Issue by yrp604** [Vector35/binaryninja-api#2691]
3055

**Describe the bug**
```
  16 @ 00000271  uncopied_bytes = _copy_from_user(to: chunk + 0x20, from: &chunk_data->str_len, n: 8)
...
  19 @ 0000028e  if (copy_size u> 0x20 || (copy_size u<= 0x20 && chunk->__offset(0x20).q u> 0x10))
```

When used as a function argument, `chunk + 20` is not written as `__offset(0x20)`. When used in an if statement, it is.

**To Reproduce**
1. Go to 00000271 in hlil graph
2. Note the copy_from_user call @ 16
3. Note the if statement @ 19

**Expected behavior**
I expect structure members to show up uniformly regardless of the context they're used in

**Version and Platform (required):**
 - Binary Ninja: Dev 2.4.3055
 - OS: Windos
 - Version 11

**Additional context**
[struct-member-arg - Copy.bndb.txt](https://github.com/Vector35/binaryninja-api/files/7325364/struct-member-arg.-.Copy.bndb.txt)

**Comment by plafosse** [comment]
This bug is likely related to: #2461

**Comment by CouleeApps** [comment]
Should be fixed in >= 3.1.3536-dev.
